### PR TITLE
#165954089 chore(setup): setup development

### PR DESCRIPTION
**Pull request by TamunoibiAprekuma**
This PR sets up testing environment

[finishes #165954089]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# Banka
\ No newline at end of file
+# Banka
+[![Build Status](https://travis-ci.com/TamunoibiAprekuma/Banka-1.svg?branch=develop)](https://travis-ci.com/TamunoibiAprekuma/Banka-1)
+
+[![Coverage Status](https://coveralls.io/repos/github/TamunoibiAprekuma/Banka/badge.svg?branch=master)](https://coveralls.io/github/TamunoibiAprekuma/Banka?branch=master)
```
Comment (Akinmyde) [inline README.md]: Why is the branch set to master

File: README.md [hunk 2]
```diff
@@ -1 +1,6 @@
-# Banka
\ No newline at end of file
+# Banka
+[![Build Status](https://travis-ci.com/TamunoibiAprekuma/Banka-1.svg?branch=develop)](https://travis-ci.com/TamunoibiAprekuma/Banka-1)
```
Comment (Akinmyde) [inline README.md]: Why is the branch set to develop

File: README.md (hunk 1 above)
Comment (TamunoibiAprekuma) [reply README.md]: I do not really understand the question. I should make the branch develop?

File: README.md (hunk 2 above)
Comment (TamunoibiAprekuma) [reply README.md]: I do not really understand the question

**Pull request merged**